tests/test_dns_cli.py

```diff
@@ -0,0 +1,187 @@
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+import boto3
+from botocore.stub import Stubber
+import pytest
+from click.testing import CliRunner
+from moto import mock_acm
+from moto import mock_route53
+from moto import mock_iam
+
+from commands.dns_cli import add_records, check_for_records, create_hosted_zone, check_r53, create_cert, check_domain, assign_domain
+
+
+@pytest.fixture(scope="function")
```

Comment:
Might be worth moving these fixtures out into a conftest.py file, now that multiple test files are using them

Reply:
yep, thats worth doing